src/shared/lib/MutationTableDownloadDataFetcher.ts

```diff
@@ -86,6 +94,17 @@ export class MutationTableDownloadDataFetcher implements ILazyMobXTableApplicati
         }
     }
 
+    private async fetchAllGenomeNexusFunctionalImpactData()
+    {
+        if (this.mutationData.result)
```

Comment:
these fetch functions should really take mutationData as a parameter instead of accessing it from instance.  they really ought to just be functions instead of methods.